src/GraphQLSchemaLinter.php

```diff
@@ -79,7 +79,12 @@ public function getVersion() {
     if (!empty($this->dependencyVersions)) {
 
       foreach ($this->dependencyVersions as $name => $required) {
-        list($err, $dep_version, $stderr) = exec_manual('cd .arcanist 2>&1 >/dev/null && npm list --depth=0 2>/dev/null | grep "%C" | cut -d "@" -f 3 | tr -d "[:space:]"', $name);
+        $bin_root = dirname($this->getDefaultBinary());
+        list($stdout) = new ExecFuture('npm list -s --depth=0 --json || exit 0')
+          ->setCWD($bin_root)
+          ->resolvex();
```

Comment:
I actually just found the `resolveJSON()` method, too, which directly returns parsed JSON.